src/Online_Bookstore/ReviewSection/ReviewedBooks.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import ReviewCard from "./ReviewCard";
+const ReviewedBooks = () => {
+  const [reviewData, setReviewData] = useState([]);
+  const navigate = useNavigate();
+  const UserData = JSON.parse(window.localStorage.getItem("user_login"));
+  const ReviewBooksData = JSON.parse(
```

Comment:
your code will break if it not get the data from local store.

Reply:
instead you can use 
const ReviewBooksData =
    JSON.parse(window.localStorage.getItem("reviewBooks")) || {};